### PR TITLE
Fix TypeError when use Blueprint.group() to group blueprint with defa…

### PR DESCRIPTION
**Pull request by chenjr0719**
…ult url_prefix, Use os.path.normpath to avoid invalid url_prefix like api//v1

### CODE REVIEW

File: sanic/blueprints.py [hunk 1]
```diff
@@ -54,7 +55,10 @@ def chain(nested):
                     yield i
         bps = []
         for bp in chain(blueprints):
+            if bp.url_prefix is None:
+                bp.url_prefix = ''
             bp.url_prefix = url_prefix + bp.url_prefix
+            bp.url_prefix = os.path.normpath(bp.url_prefix)
```
Comment (yunstanford) [inline sanic/blueprints.py]: I don't think it should call normpath here.

Comment (vltr) [reply sanic/blueprints.py]: @yunstanford there's a discussion regarding this on #1325, @chenjr0719 just needs to update the PR afaik.

Comment (chenjr0719) [reply sanic/blueprints.py]: @vltr @yunstanford I just remove `normpath` to make this PR simple and focus on the original issue about default value of `url_prefix`.

**Comment by chenjr0719** [comment]
@ahopkins Oops, my bad. The css file is created by `pytest-html` which I used to trace the output and log of unit test. I forgot to clean it up.

**Pull request merged**